mmv1/templates/terraform/examples/firewall_policy_rule.tf.tmpl

```diff
@@ -49,3 +49,20 @@ resource "google_compute_firewall_policy_rule" "{{$.PrimaryResourceId}}" {
     }
   }
 }
+
+resource "google_tags_tag_key" "basic_key" {
+  description = "For keyname resources."
+  parent      = "organizations/{{index $.TestEnvVars "org_id"}}"
+  purpose     = "GCE_FIREWALL"
+  short_name  = "{{index $.Vars "tag_key"}}"
```

Comment:
Failed to find tag_key environment variable defined in YAML file when validating the file templates/terraform/examples/firewall_policy_rule.tf.tmpl. Please define this in vars